php/src/Docker/DockerActionManager.php

```diff
@@ -897,11 +924,23 @@ public function ConnectMasterContainerToNetwork() : void
         $this->ConnectContainerIdToNetwork('nextcloud-aio-mastercontainer', '');
         // Don't disconnect here since it slows down the initial login by a lot. Is getting done during cron.sh instead.
         // $this->DisconnectContainerFromBridgeNetwork('nextcloud-aio-mastercontainer');
+
+	$apacheContainerAdditionalNetwork = $this->configurationManager->GetApacheContainerAdditionalNetwork();
+        if ($apacheContainerAdditionalNetwork !== '') {
+            $this->ConnectContainerIdToExistingNetwork('nextcloud-aio-mastercontainer', '', $apacheContainerAdditionalNetwork);
+	}
     }
 
     public function ConnectContainerToNetwork(Container $container) : void
     {
         $this->ConnectContainerIdToNetwork($container->GetIdentifier(), $container->GetInternalPort());
+
+        if ($container->GetIdentifier() === 'nextcloud-aio-apache') {
+            $apacheContainerAdditionalNetwork = $this->configurationManager->GetApacheContainerAdditionalNetwork();
+            if ($apacheContainerAdditionalNetwork !== '') {
+                $this->ConnectContainerIdToExistingNetwork($container->GetIdentifier(), $container->GetInternalPort(), $apacheContainerAdditionalNetwork);
```

Comment:
Can we maybe simply re-use the ConnectContainerIdToNetwork function?

Possibly something like this? ConnectContainerIdToNetwork(nextcloud-aio-apache, $internalPort, $apacheContainerAdditionalNetwork)

Reply:
I intentionally separated it out because `ConnectContainerIdToNetwork()` also tries to create the network if it does not exist. 
When someone's using this feature, the network should be created externally and must already exist. If it does not exist, it is likely a typo or incorrect usage, and AIO should throw an error instead of silently creating a new network and proceeding.

If code duplication is a concern I can update the `ConnectContainerIdToNetwork()` to take an optional parameter `alsoCreateNetwork = true` with true as default value, and set to false when trying to connect to this feature.

Reply:
I see but isn't it bad if the process fails? Will not people create addiitonal support requests in that case? I would probably rather go the way to also create the network if it does not exist instead of failing out with the last container...